src/main/java/mate/academy/booksstore/mapper/MapperUtil.java

```diff
@@ -0,0 +1,24 @@
+package mate.academy.booksstore.mapper;
+
+import java.util.Set;
+import java.util.stream.Collectors;
+import mate.academy.booksstore.model.Category;
+import mate.academy.booksstore.repository.CategoryRepository;
+import org.mapstruct.Named;
+import org.springframework.stereotype.Component;
+
+@Component
+public class MapperUtil {
```

Comment:
Delete this class. We do not need to go to DB. Simply create Category objects with Ids in @AfterMapping method.